README.md

```diff
@@ -82,9 +82,13 @@ end
 ```
 
 ## Installation
+you can use it either downloading with curl,
```

Comment:
Sure thing, I think a patch version bump would be appropriate. Pre 1.0, it doesn't really matter :wink: